docs/querying/basics.md

```diff
@@ -229,7 +229,7 @@ The same works for range vectors. This returns the 5-minute rate that
 
     rate(http_requests_total[5m] @ 1609746000)
 
-`@` modifier supports all representation of float literals described
+The `@` modifier supports all representation of float literals described
 above within the limits of `int64`. It can also be used along
 with `offset` modifier where the offset is applied relative to the `@`
```

Comment:
…with the `offset` modifier…

(I know, not a new issue, but let's get more consistent with articles, now that we are on it.)